cypress/integration/hostedWorkflows.js

```diff
@@ -68,15 +65,15 @@ describe('Dockstore my workflows', function() {
           .get('#saveNewVersionButton')
           .click()
 
-        // Should have a version 0
```

Comment:
In the future, maybe this should be a constant in case we change our mind again. 
Not a big deal now